catalog.html

```diff
@@ -83,77 +83,79 @@ <h2 class="selected-products">Сливочное мороженое</h2>
 
       <section class="catalog-products">
         <h2 class="visually-hidden">Фильтр.</h2>
-        <h3 class="catalog-filter-title">Сортировка</h3>
-        <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
-          <select class="select-control" aria-label="Сортировка">
-            <option value="popular">по популярности</option>
-          </select>
-          <fieldset class="catalog-filter-group">
-            <legend class="catalog-filter-title">Жирность</legend>
-            <ul class="catalog-filter-list">
-              <li class="catalog-filter-item">
-                <label class="control">
-                  <input class="control-input" type="radio" name="product-group" value="0%">
-                  0%
-                </label>
-              </li>
-              <li class="catalog-filter-item">
-                <label class="control">
-                  <input class="control-input" type="radio" name="product-group" value="10%" checked>
-                  10%
-                </label>
-              </li>
-              <li class="catalog-filter-item">
-                <label class="control">
-                  <input class="control-input" type="radio" name="product-group" value="30%">
-                  30%
-                </label>
-              </li>
-              <li class="catalog-filter-item">
-                <label class="control">
-                  <input class="control-input" type="radio" name="product-group" value=">30%">
-                  выше 30%
-                </label>
-              </li>
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Сортировка</legend>
+          <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
```

Comment:
Здесь идут валидационные ошибки:
1. Тег `form` не закрыт в пределах родителя
2. А вот нижний `</form>` наоборот не открыт по идее

Но все это идет от того, что нарушен порядок тегов. `fieldset` по-хорошему должен быть внутри формы, поэтому тут нужно поместить его внутрь

```suggestion
<form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
  <fieldset class="catalog-filter-group">
    <legend class="catalog-filter-title">Сортировка</legend>
      <select class="select-control" aria-label="Сортировка">
        <option value="popular">по популярности</option>
      </select>
  </fieldset>
  
  ....
</form>
```